Source/ThirdParty/Uri.js

```diff
@@ -262,6 +262,9 @@
 			result += this.scheme + ':';
 		if (this.authority)
 			result += '//' + this.authority;
+		else
+			if (this.scheme == "file")
+				result += '//';
```

Comment:
This isn't technically in our code-base, but I would change this to include curly braces to match our style guide https://github.com/CesiumGS/cesium/blob/master/Documentation/Contributors/CodingGuide/README.md#formatting

Reply:
I would actually recommend we maintain the style of the outer code (there are no curly braces elsewhere in this file).

Normally we do not modify ThirdParty code, but if this does fix a real issue and has no impact elsewhere, then I'm okay with this change (haven't really looked into it yet).  But we should have a comment here stating it's a modification from the original with a link back to this PR.

Reply:
I can add the statement, as well as braces, if you like.

Reply:
Is it also worth a Changes.md entry? I could add that as well.